packages/p2p/package.json

```diff
@@ -34,6 +34,8 @@
         "@deriv/hooks": "^1.0.0",
         "@deriv/shared": "^1.0.0",
         "@deriv/stores": "^1.0.0",
+        "@deriv/api": "^1.0.0",
```

Comment:
@niloofar-deriv It seems both `StoreProvider` and `APIProvider` are missing in the `App` component 👀

Reply:
As we talked we can ignore this here!